cluster-autoscaler/main.go

```diff
@@ -317,7 +321,7 @@ func createAutoscalingOptions() config.AutoscalingOptions {
 	}
 }
 
-func getKubeConfig() *rest.Config {
+func getKubeConfig(context config.AutoscalingOptions) *rest.Config {
```

Comment:
`context` param is not used after the changes, so let's drop it.

Reply:
done @x13n